gp-includes/routes/project.php

```diff
@@ -386,8 +390,9 @@ public function permissions_post( $project_path ) {
 				'locale_slug' => gp_post( 'locale' ),
 				'set_slug'    => gp_post( 'set-slug' ),
 			) );
-			if ( $this->invalid_and_redirect( $new_permission, gp_url_current() ) ) return;
-			$permission = GP::$validator_permission->create( $new_permission );
+			if ( $this->invalid_and_redirect( $new_permission, gp_url_current() ) ) { return;
+			}
```

Comment:
```suggestion
			if ( $this->invalid_and_redirect( $new_permission, gp_url_current() ) ) {
				return;
			}
```

Reply:
I have this fix in a commit for `Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace`
Should I push to this same PR all the wpcs Squiz.(...) and WordPress.(...) sniff fixes?

Reply:
No, but changes like this should not introduce new code style issues. Please only adjust the lines changed in this PR.

Reply:
Ok, fixing it.